src/ResourceManagement/RecoveryServices/Microsoft.Azure.Management.RecoveryServices/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "4.1.0-preview",
+  "version": "4.2.0-preview",
```

Comment:
@dragonfly91 can you update AssemblyInfo.cs files.
Soon we are moving to new csproj based project system and this version will be used, so please update it to reflect the update version from project.json